cif_pow.dic

```diff
@@ -7418,6 +7418,75 @@ save_PD_INSTR_DETECTOR
     _name.object_id               PD_INSTR_DETECTOR
     _category_key.name            '_pd_instr_detector.id'
 
+    loop_
+      _description_example.case
+      _description_example.detail
+;
+        loop_
+        _pd_instr_detector.id
+        _pd_instr.dist_spec_detc
+        _pd_instr.slit_ax_spec_detc
+        _pd_instr.slit_eq_spec_detc
+        _pd_instr_detector.instr_id
+            fd343493   320   8.5   0.075   b4131be5
+            47540661   320   8.5   0.075   b4131be5
+;
+;
+        These detectors are paired with the instrument identified by the
+        id b4131be5.
+
+        There are two detectors, fd343493 and 47540661. Both detectors are
+        positioned 320 mm from the specimen, and are defined by two slits,
+        8.5 mm in the axial direction, and 0.075 mm in the equatorial direction.
+        As the instrument description says that this instrument has 'Dual strip
+        detectors', it can be assumed that the detector window is 8.5 mm wide,
```

Comment:
"It can be assumed..."  Does this mean there is information missing that has to be guessed? The statement that we are using a strip detector is only human-readable, which doesn't help a programmer who wants to reliably calculate values. Or is the statement about the detector window just for explanatory purposes, so could be rephrased as "As the detector window is 8.5mm wide, with 75 micron wide pixels, `_pd_instr.slit_eq_spec_detc` is 0.075 and _pd_instr.slit_ax_spec_detc is 8.5"?

Reply:
That sounds better. I'll rephrase.